packages/app/src/cli/services/webhook/trigger-flags.ts

```diff
@@ -166,51 +102,7 @@ function deliveryMethodInstructions(method: string): string[] {
   return []
 }
 
-/**
- * Check if the address is valid and return a valid address - method pair
- *
- * @param address - Address
- * @returns [address, deliveryMethod]
- */
-export function parseAddressFlag(address: string): [string, string] {
-  const method = deliveryMethodForAddress(address)
-  if (method === undefined) {
-    throw new AbortError(
-      'No delivery method available for the address',
-      `Use a valid address for ${DELIVERY_METHOD.HTTP}, ${DELIVERY_METHOD.PUBSUB} or ${DELIVERY_METHOD.EVENTBRIDGE}`,
-    )
-  }
-
-  return [address.trim(), method]
-}
-
-/**
- * Infer the delivery method from address
- *
- * @param address - Address
- * @returns deliveryMethod or undefined
- */
-export function deliveryMethodForAddress(address: string): string | undefined {
```

Comment:
Does this mean that not needing to pass the delivery method (because it was inferred from the address) is not a possibility anymore? 
or is it for when there're no flags?

The change looks good to me, just trying to understand

Reply:
Oops, good catch! Without flags, we were always prompting for the delivery method, but I didn't realize it was being inferred when the address is passed as a flag... Let me restore that functionality.

Reply:
Should be fixed by https://github.com/Shopify/cli/pull/3727/commits/e959cb9696876c2ca06a5f44e63da30c9f06cf8a